OPConsultation-output.json

```diff
@@ -0,0 +1,2874 @@
+            }
+          ],
+          "text": "Assessment of pregnancy status prior to imaging"
+        },
+        "subject": {
+          "reference": "urn:uuid:c4d052b5-2d9f-4ebf-b617-764efffa08de",
+          "display": "Testing ABDM NCG",
+          "type": "Patient"
+        },
+        "effectiveDateTime": "2024-08-06T12:18:11+05:30",
+        "performer": [
+          {
+            "reference": "urn:uuid:41295111-04f9-4b83-b186-ef2975db1c7e",
+            "type": "Practitioner"
+          }
+        ],
+        "valueString": "Not pregnant"
+      }
+    },
+    {
+      "fullUrl": "urn:uuid:320b2b02-1872-484b-907f-f7d0d873fac0",
+      "resource": {
+        "resourceType": "Observation",
+        "id": "320b2b02-1872-484b-907f-f7d0d873fac0",
+        "status": "final",
+        "code": {
+          "coding": [
+            {
+              "code": "80182007",
+              "system": "http://snomed.info/sct",
+              "display": "Irregular periods"
+            }
+          ],
+          "text": "Irregular periods"
+        },
+        "subject": {
+          "reference": "urn:uuid:c4d052b5-2d9f-4ebf-b617-764efffa08de",
+          "type": "Patient"
+        },
+        "effectiveDateTime": "2024-08-06T12:18:11+05:30",
+        "performer": [
+          {
+            "reference": "urn:uuid:41295111-04f9-4b83-b186-ef2975db1c7e",
+            "type": "Practitioner"
+          }
+        ],
+        "valueString": "Irregular periods"
+      }
+    },
+    {
+      "fullUrl": "urn:uuid:1af9fa5c-28cb-4c45-8d0a-24002faddeb8",
+      "resource": {
+        "resourceType": "ServiceRequest",
+        "id": "1af9fa5c-28cb-4c45-8d0a-24002faddeb8",
+        "status": "active",
+        "intent": "order",
+        "code": {
+          "text": "Chest CT"
+        },
+        "subject": {
+          "reference": "urn:uuid:c4d052b5-2d9f-4ebf-b617-764efffa08de",
+          "display": "Testing ABDM NCG",
+          "type": "Patient"
+        },
+        "occurrenceDateTime": "2013-05-08T09:33:27+07:00",
+        "requester": {
+          "reference": "urn:uuid:Practitioner/example1"
+        }
+      }
+    },
+    {
+      "fullUrl": "urn:uuid:fc980123-6830-45ca-85e7-36ba272e53b0",
+      "resource": {
+        "resourceType": "ServiceRequest",
+        "id": "fc980123-6830-45ca-85e7-36ba272e53b0",
+        "status": "active",
+        "intent": "order",
+        "code": {
+          "text": "Thyroxine (T4) free [Mass/​volume] in Serum or Plasma"
+        },
+        "subject": {
+          "reference": "urn:uuid:c4d052b5-2d9f-4ebf-b617-764efffa08de",
+          "type": "Patient"
+        },
+        "occurrenceDateTime": "2015-08-27T09:33:27+07:00",
+        "requester": {
+          "reference": "urn:uuid:Practitioner/example2"
+        }
+      }
+    },
+    {
+      "fullUrl": "urn:uuid:af32435a-040f-435b-995e-81d9a7d36823",
+      "resource": {
+        "resourceType": "MedicationStatement",
```

Comment:
Change to some other drug! Not representative again. How can the patient is taking Amox and yet the doctor is again prescribing amox? You can say the medicationStatement.status = completed. That indicates that the patient was taking the medication, and then the doctor is again prescribing amox through medicationRequest